components/prospects/ConfirmModal.tsx

```diff
@@ -0,0 +1,94 @@
+import { Modal, Box, Typography, Button } from '@mui/material';
+
+function ConfirmModal({
```

Comment:
this file shouldn't be inside /prospects, but in a folder helper in order to be reused whenever we want to confirm an action